tests/test_ext_intersphinx.py

```diff
@@ -229,3 +234,53 @@ def test_getsafeurl_unauthed():
     expected = 'https://domain.com/project/objects.inv'
     actual = _get_safe_url(url)
     assert expected == actual
+
+
+def test_debug_noargs(capsys):
+    """debug interface, without arguments"""
+    with pytest.raises(SystemExit):
+        debug(['sphinx/ext/intersphinx.py'])
+
+    expected = dedent("""
+        Print out an inventory file.
+        Error: must specify local path or URL to an inventory file.
+    """).strip()
+    stdout, stderr = capsys.readouterr()
+    assert stdout == ""
+    assert stderr == expected + "\n"
```

Comment:
I don't like so much that changing message causes test failure. It's not essentially of testing, and it's boring works.
But it's not big problem. Okay, go ahead.